src/components/languageMenu/LanguageMenuPure.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import OutsideClickHandler from 'react-outside-click-handler'
+import LanguageContext from 'src/contexts/language'
+import './language-menu.less'
+import configPages from 'src/config'
+
+const LanguageMenuPure = ({
+  closeLanguageMenu,
+  toggleLanguageMenu,
+  isLanguageMenuShown
+}) => {
+  const locationPathname = window.location.href
+    .split('/')
+    .filter(function(element) {
+      return element !== ''
+    })
+    .pop()
+  if (
+    configPages[locationPathname] &&
+    configPages[locationPathname].languages !== -1
+  ) {
+    const currentLanguage = LanguageContext._currentValue
+    const currentAvailableLanguage = configPages[locationPathname].languages
+    const domainHref = 'https://platinum.fund/'
+
+    const currentLanguageContent = (
```

Comment:
Could you please move it outside of the function, i.e. make it as a React component. And probably move to another file in the same directory